src/openforms/js/components/admin/form_design/variables/types.js

```diff
@@ -8,6 +8,7 @@ const Variable = PropTypes.shape({
   source: PropTypes.string,
   prefillPlugin: PropTypes.string,
   prefillAttribute: PropTypes.string,
+  prefillIdentifierRole: PropTypes.string,
```

Comment:
```suggestion
  prefillIdentifierRole: PropTypes.oneOf(['main', 'authorised_person']),
```
we can make this stricter, likely we also should include the empty string for existing components.